ReleaseChecklist.md

```diff
@@ -62,9 +73,14 @@
  - [ ] Create a tag using ``git tag --annotate v$VERSION`` and push it with ``git push --tags``.
 
 ### Post-release
- - [ ] Publish the blog post.
+ - [ ] Make sure the documentation for the new release has been published successfully.
+       Go to https://readthedocs.org/projects/solidity/ and verify that the new version is listed, works and is marked as default.
```

Comment:
```suggestion
       Go to [readthedocs/solidity](https://readthedocs.org/projects/solidity/) and verify that the new version is listed, works and is marked as default.
```

Reply:
Done. Just used a slightly different text for the link.